test/server/chat-plugins/youtube.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Tests for the Youtube room plugin.
+ * Written by mia-pi.
+ */
+'use strict';
+const YoutubeInterface = require('../../../.server-dist/chat-plugins/youtube').YoutubeInterface;
+const assert = require('../../assert').strict;
+const Config = require('../../../.server-dist/config-loader').load();
```

Comment:
```suggestion
const assert = require('../../assert').strict;
```
Config is already set up in `test/main.js`, so requiring it again is unnecessary, and moreover resets `Config.nofswriting`, which caused the tests to fail.